### PR TITLE
Adding note about syntax for top declaration when using es6

### PR DESCRIPTION
**Pull request by acolchado**
@wruth This documentation note closes your question in issue #26 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,19 @@ injector().inject(function(TasksPrinter){
   TasksPrinter.print();
 });
 ```
+
+##### Usage note for ES6 syntax
+
+While it is tempting to utilize the fat arrow syntax in this top declaration file like the example below, it will not be supported by spur-ioc. For more information, read issue [#26](https://github.com/opentable/spur-ioc/issues/26). Instead use the recommended approach above. There isn't a compelling reason to add that additional support. If you use this style, it will break as the report in issue #26.
+
+```javascript
+var injector = require("./injector");
```
Comment (dgdblank) [inline README.md:130]: Is this supposed to be using es6 syntax? Should you be using `import injector from ./injector`?

**Review by dgdblank — COMMENTED** [review]
Quick question about syntax.

File: README.md (hunk 1 above)
Comment (acolchado) [reply README.md:130]: Only for the function definition. Imports are not supported in most versions of node, so I kept it as close to what is supported on most versions.

Comment (dgdblank) [reply README.md:130]: LGTM.

**Review by dgdblank — APPROVED** [review]

**Pull request merged**